Makefile

```diff
@@ -33,6 +33,22 @@ update-beats:
 	@$(MAKE) update
 	@echo --- Use this commit message: Update beats framework to `cat vendor/vendor.json | python -c 'import sys, json; print([p["revision"] for p in json.load(sys.stdin)["package"] if p["path"] == "github.com/elastic/beats/libbeat/beat"][0][:7])'`
 
+.PHONY: check-headers
+check-headers:
+ifndef CHECK_HEADERS_DISABLED
+	@go get -u github.com/elastic/go-licenser
+	@go-licenser -exclude x-pack
+	@go-licenser -license Elastic x-pack
+endif
+
+.PHONY: add-headers
+add-headers:
+ifndef CHECK_HEADERS_DISABLED
+	@go get -u github.com/elastic/go-licenser
+	@go-licenser -exclude x-pack
+	@go-licenser -license Elastic x-pack
+endif
+
```

Comment:
What do you think about adding a make target for building the x-pack version?  something like:

```
${BEAT_NAME}.x-pack: $(GOFILES_ALL) ## @build build the x-pack enabled version
	go build -o $@ $(GOBUILD_FLAGS) ./x-pack/${BEAT_NAME}
```